lms/static/js/verify_student/models/reverification_model.js

```diff
@@ -1,51 +0,0 @@
-/**
```

Comment:
Note to reviewers: this functionality has been folded into `verification_model.js`, which now handles both cases.